integration/apps/rails-five/config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  get 'health', to: 'health#check'
+
+  # Basic test scenarios
+  get 'basic/default', to: 'basic#default'
+  get 'basic/fibonacci', to: 'basic#fibonacci'
```

Comment:
```suggestion
  # Basic test scenarios
  get '/', to: 'basic#default'
  get 'basic/default', to: 'basic#default'
  get 'basic/fibonacci', to: 'basic#fibonacci'
```

To match my suggestion on the rack side, I suggest having `/` be something with a bit of feedback by default :)

Reply:
Yes, good idea, thank you.